.github/workflows/bump-elastic-stack.yml

```diff
@@ -31,14 +30,9 @@ jobs:
         with:
           ref: ${{ matrix.branch }}
 
-      - name: Setup Git
-        uses: elastic/apm-pipeline-library/.github/actions/setup-git@current
-
-      - name: Install Updatecli in the runner
-        uses: updatecli/updatecli-action@453502948b442d7b9a923de7b40cc7ce8628505c
-
-      - name: Run Updatecli
-        env:
-          GITHUB_TOKEN: ${{ github.token }}
-          BRANCH: ${{ matrix.branch }}
```

Comment:
The `BRANCH` environment variable isn't defined anymore.

Reply:
Oh dear! C&P didn't help here.

I need to figure out if `env` can be used within the context of the `uses`

Reply:
https://github.com/elastic/apm-server/pull/10251/commits/39ac347deadf5a468c41dc64ab08def78711fffa